plaidml/bridge/openvino/ops/one_hot.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <iostream>
+
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+
+#include "ngraph/opsets/opset1.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace PlaidMLPlugin {
+
+void registerOneHot() {
+  registerOp("onehot", [](const Context& ctx) {
+    IE_ASSERT(ctx.operands.size() == 4);
+
+    auto indices = ctx.operands.at(0);
+    auto depth = get_shape_from_constant_operand(1, ctx.layer);  // ctx.operands.at(1);
```

Comment:
The comment at the end of the line should be dropped (or there should be an additional comment about it)